google/cloud/aiplatform/metadata/context.py

```diff
@@ -57,3 +61,20 @@ def _update_resource(
         cls, client: utils.MetadataClientWithOverride, resource: proto.Message,
     ) -> proto.Message:
         return client.update_context(context=resource)
+
+    def add_artifacts_or_executions(
```

Comment:
add_artifacts_and_executions? instead of or?

Reply:
done